tensorbay/client/tests/test_cloud_storage.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
Actually, you need to add some integration tests to ensure that the function works.